docs/guide/introduction/quickstart.md

```diff
@@ -0,0 +1,77 @@
+# Quickstart
+
+## Initial setup
+
+To start using EasyP, you need to initialize the project.
+
+To do this, run the following command in the terminal:
+
+```bash
+easyp init 
+```
+
+## Setup linter rules
+
+For example, you can use default linter rules.
+
+```yaml
+lint:
+  use:
+    - DEFAULT
+```
+
+After setting up the linter rules, you can run the linter command:
+
+```bash
+easyp lint
+```
+
+## Setup package dependencies
+
+If you ara using third-party packages,
+you need to add them to the `dependencies` section in the `easyp.yaml` file.
+
+```yaml
+lint:
+  use:
+    - DEFAULT
+deps:
+  - github.com/googleapis/googleapis
+  - github.com/grpc-ecosystem/grpc-gateway@v2.20.0
+```
+
+And now you can download packages by command
+
+```bash
+easyp mod download
```

Comment:
`download` command will download deps from your `easyp.lock` file.
If lock file is missed then easyp download packages with versions from `easyp.yaml` file and create lock file

`update` command ignores lock file and download packages with versions from `easyp.yaml` file and create lock file with them.